.github/workflows/lint.yml

```diff
@@ -45,18 +48,25 @@ jobs:
             experimental/node_modules
             */*/node_modules
             experimental/*/*/node_modules
-          key: lint-${{ matrix.package-group }}-${{ runner.os }}-${{ matrix.node_version }}-${{ hashFiles('**/package.json') }}-20211009.2
+          key: lint-${{ matrix.package-group }}-${{ runner.os }}-${{ matrix.node_version }}-${{ hashFiles('**/package.json') }}-20211216.2
 
       # On a cache miss, install dependencies
       - name: Bootstrap
         if: steps.cache.outputs.cache-hit != 'true'
         working-directory: ${{ matrix.package-group }}
         run: |
           npm install --ignore-scripts
-          npx lerna bootstrap --no-ci --ignore-scripts --hoist --nohoist='zone.js' -- --only=dev
+          npx lerna bootstrap --no-ci --hoist --nohoist='zone.js'
 
       - name: Lint
         working-directory: ${{ matrix.package-group }}
         run: |
           npm run lint
           npm run lint:examples
+
+      - name: Lint doc files
+        if: ${{ matrix.package-group == '.' }}
```

Comment:
Not running on experimental packages?

Reply:
Experimental packages are already included in typedoc config in root `tsconfig.json`. See .github/workflows/docs.yaml for reference.